android/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,5 @@
     CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -->
 
-<manifest
-    package="com.facebook.reactnative.androidsdk">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
this is a non-backwards-compatible change and not the way I've done it in other modules